frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -197,26 +200,26 @@ const CsvSchemaDocumentation = () => {
               </p>
             </li>
             <li className="usa-process-list__item">
-              <h4 className="usa-process-list__heading">Enter your data</h4>
+              <h3 className="usa-process-list__heading">Enter your data</h3>
               <p>
                 Following the spreadsheet guidelines below, enter properly
                 formatted values in the relevant fields. Some fields require
                 data, while others don’t.
               </p>
             </li>
             <li className="usa-process-list__item">
-              <h4 className="usa-process-list__heading">
+              <h3 className="usa-process-list__heading">
                 Export or save your data
-              </h4>
+              </h3>
               <p>
                 Make sure your spreadsheet is in a CSV format. SimpleReport
                 doesn’t accept XLS, XLXS, or any other formats.
               </p>
             </li>
             <li className="usa-process-list__item">
-              <h4 className="usa-process-list__heading">
+              <h3 className="usa-process-list__heading">
                 Use the uploader on SimpleReport
-              </h4>
+              </h3>
               <p>
                 Visit the{" "}
                 <LinkWithQuery to={"/results/upload"}>
```

Comment:
looks like another bad link in our doc :(

Reply:
great catch, fixed